src/tools/std20/ranges/access/test/cbegin.test.cpp

```diff
@@ -0,0 +1,57 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/ranges.hpp"
+
+// other includes
+#include <array>
+#include <deque>
+#include <forward_list>
+#include <list>
+#include <set>
+#include <string>
+#include <string_view>
+#include <vector>
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+// test code
+struct Foo {};
+
+SCENARIO( "ccbegin" ) {
```

Comment:
"ccbegin" -> "cbegin"

Reply:
That has been corrected - see fix/review